src/main/java/me/totalfreedom/totalfreedommod/command/Command_orbit.java

```diff
@@ -11,8 +11,7 @@
 import org.bukkit.util.Vector;
 
 @CommandPermissions(level = Rank.ADMIN, source = SourceType.BOTH)
-@CommandParameters(description = "POW!!! Right in the kisser! One of these days Alice, straight to the Moon - Sends the specified player into orbit.",
-        usage = "/<command> <target> [<<power> | stop>]")
+@CommandParameters(description = "Sends the specified player into orbit.", usage = "/<command> <player> [<power> | stop]")
```

Comment:
Think I liked this one how it was before... I like a bit of Madgeek's descriptions dotted around in there.

Reply:
Alright, I can revert that